packages/ag-charts-enterprise/src/polar-axes/radius-number/radiusNumberAxis.ts

```diff
@@ -202,7 +202,9 @@ export class RadiusNumberAxis extends _ModuleSupport.PolarAxis {
 
     normaliseDataDomain(d: number[]) {
         const { min, max } = this;
-        return normalisedExtent(d, min, max);
+        const domain = normalisedExtent(d, min, max);
+
+        return { domain, clipped: domain.clipped };
```

Comment:
Why in some of the places you create a copy of the domain array and in others (such as this) you don't?

Reply:
The need for the spread is now redundant, so I've cleaned this up.